Makefile.devel

```diff
@@ -89,14 +89,14 @@ DEVELOPERS=\
 # PRERELEASE_PREFIX is used to match PRERELEASE_TAG in sed
 # line in tag target below, which edits $(DOWNLOADHTML)
 # The prereltag.txt is kept as a record in the distrib area
-# of the current pre-release version (currently not used explicitly 
+# of the current pre-release version (currently not used explicitly
 # anywhere for web pages/whatever).
-PRERELEASE_PREFIX=4\.4\.1~pre
-PRERELEASE_TAG=4.4.1~pre
+PRERELEASE_PREFIX=4\.4\.1-pre
```

Comment:
Hi @zhenya1007, thanks for your suggestion. Indeed, `4.4.1~pre` won't work out of the box with function `version-to-list`. But on the other hand, this issue with the tilde is not that critical currently, as the suffix `~pre` is not intended to be kept for PG releases: it was just added as a "SNAPSHOT" qualifier within the Git repo. However, according to the default value displayed in the documentation of `(describe-variable 'version-regexp-alist)`, a `-git` suffix would actually be preferable for that usage...
Thus, for the moment (to ensure the monotonicity of the version number) I propose you revert that change back to `4.4.1~pre` in the `Makefile.devel` file, and I shall apply this new suffix `-git` after the 4.5 version is released (namely, switching to something like `4.5.1-git`).